e2e/cypress/integration/menus/menu_actions_spec.js

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @menu
+
+describe('LHS Menu action', () => {
+    before(() => {
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        cy.apiCreateTeam('team', 'Team');
+    });
+
+    it('MM-T1669 Team icon shows active state on click', () => {
+        // # Assert if the team wrapper in LHS present
```

Comment:
Please use `*` on all assertion/verification comments.

```suggestion
        // * Assert if the team wrapper in LHS present
```